scripts/utils/starknet.py

```diff
@@ -201,7 +206,10 @@ def dump_deployments(deployments):
 
 
 def get_deployments():
-    return json.load(open(DEPLOYMENTS_DIR / "deployments.json", "r"))
+    try:
+        return json.load(open(DEPLOYMENTS_DIR / "deployments.json", "r"))
+    except FileNotFoundError:
+        return {}
```

Comment:
Is this for when the devnet is dockerized?

Reply:
in some cases, you may want to have in your deployment script a check regarding previously deployed contracts. However, when there are no previous deployments, this would raise because no file exist.